src/components/Card/CardGrid.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+
+import "./CardGrid.css";
+
+const CardGrid = (props: any) => {
```

Comment:
class instead of function?

can we give props a type that just includes children?

Reply:
Same about classes vs functions. I'm gonna create the Props interface